jekyll/_cci2/runner-installation.adoc

```diff
@@ -0,0 +1,390 @@
+= Installing the CircleCI Runner
+:page-layout: classic-docs
+:page-liquid:
+:icons: font
+:toc: macro
+:toc-title:
+
+This page describes how to install CircleCI Runner on Linux and macOS platforms.
+
+toc::[]
+
+=== Prerequisites
+
+==== Supported Platforms
+
+Currently, the following platforms are supported with CircleCI Runner:
+
+- Ubuntu 18.04 running on an x86_64 (AMD64) or ARM64 architecture.
+- macOS Catalina (10.15.6+) or above, running on an x86_64 architecture.
+
+CAUTION: Running in a container is not currently supported.
```

Comment:
We’ve added instructions to the preview docs for running it in a container.

Reply:
Added Docker installation content to page.

Reply:
Added to document

Reply:
Added to document